lib/stripe/types.ex

```diff
@@ -12,6 +12,30 @@ defmodule Stripe.Types do
           state: String.t() | nil
         }
 
+  @type authorization_controls :: %{
```

Comment:
What if we inlined this in a `Stripe.Issuing.Types` module?  Unless you see this type used outside the Issuing scope?.

Reply:
Yeah, I noticed that pattern in some other instances. I'm happy to move the issuing-specific types into a separate module or inline them where appropriate.

Reply:
Yeah that might be good!